ListingManager/ListingManager.cs

```diff
@@ -59,25 +60,47 @@ private static bool TryGetListing(string listingPath, out ListingInformation lis
 
                 var curListingData = listingData[i];
 
-                if (listingNumber == curListingData.ListingNumber)
+                if (chapteronly == false && byfolder == false)
+                {
+                    if (listingNumber == curListingData.ListingNumber) //default
+                    {
+                        continue;
+                    }
+                }
+
+                string completeListingNumber = listingNumber + ""; //default
+                int listingChapterNumber = curListingData.ChapterNumber; //default
+
+                if (chapteronly)
+                {
+                    completeListingNumber = curListingData.ListingNumber + curListingData.ListingSuffix + "";
+
+                }
+
+                if (byfolder)
                 {
-                    continue;
+                    listingChapterNumber = FileManager.GetFolderChapterNumber(pathToChapter);
                 }
 
-                UpdateListingNamespace(cur, curListingData.ChapterNumber, 
-                    listingNumber+"", 
+
+                UpdateListingNamespace(cur, listingChapterNumber,
+                    completeListingNumber,
                     curListingData.ListingDescription, verboseMode, preview);
 
                 if (GetPathToAccompanyingUnitTest(cur, out string pathToTest))
                 {
                     Console.Write("Updating test. ");
-                    UpdateListingNamespace(pathToTest, curListingData.ChapterNumber,
-                        listingNumber+"", 
+                    UpdateListingNamespace(pathToTest, listingChapterNumber,
+                        completeListingNumber,
                         curListingData.ListingDescription, verboseMode, preview);
                 }
+
+
+
+
```

Comment:
This method has a lot of extra whitespace throughout. Lets just get rid of all the double, triple, and quadruple? empty lines. I think I also did this somewhere else...if you see it please feel free to remove it...